### PR TITLE
Added OSConsole encapsulation class for WebSocket

### PR DESCRIPTION
**Pull request by andy-maier**
For details, see the commit message.

### CODE REVIEW

File: tests/common/http_mocked_fixtures.py
Comment (EdGue42) [inline tests/common/http_mocked_fixtures.py:221]: minor: we often resort to fixed uuids for such uris, such as 00000000-1111-2222-3333-444444444444

File: zhmcclient/_os_console.py
Comment (EdGue42) [inline zhmcclient/_os_console.py:185]: code duplication: check + raise could go into a dedicated helper method "checkConnected()"

**Review by EdGue42 — APPROVED** [review]

File: zhmcclient/_os_console.py
Comment (andy-maier) [reply zhmcclient/_os_console.py:185]: DONE
Added `_check_connected()`, and for symmetry also `_check_not_connected()`.

File: tests/common/http_mocked_fixtures.py
Comment (andy-maier) [reply tests/common/http_mocked_fixtures.py:221]: Sure, but I want to indicate in the string that this is a faked URI. We do that in many tests and in the mock support.

**Comment by andy-maier** [comment]
@EdGue42 I rejected one comment and followed the other one.
The PR needs to wait until its underlying PR is merged.

**Pull request merged**
